### PR TITLE
Polygon - **Breaking Change** the reputation commands **ip**, **domain**, **file**,  **url**, **process** and **registry**

### PR DESCRIPTION
**Pull request by orhovy**
removed indicators use
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/29016

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Polygon/Integrations/Polygon/Polygon.py [hunk 1]
```diff
@@ -321,12 +322,21 @@ def get_packages_indicators(report):
...
-        ids.append(file)
-    return ids
+        command_results.append(CommandResults(
+            readable_output=get_human_readable_analysis_info(analysis_info),
+            outputs_prefix="Polygon.Analysis",
+            outputs_key_field="ID",
+            outputs=analysis_info,
+            indicator=file,
+            raw_response=res
+            )
```
Comment (DeanArbel) [inline Packs/Polygon/Integrations/Polygon/Polygon.py:332]: Please omit analysis_info output from the indicator's output, as it's already being returned elsewhere.
These CommandResults should be just for the indicator and its relevant human readable.

File: Packs/Polygon/Integrations/Polygon/Polygon.py [hunk 2]
```diff
@@ -339,7 +349,15 @@ def get_network_indicators(report):
...
             )
         )
-        ids.append(domain)
+        command_results.append(CommandResults(
+            readable_output=get_human_readable_analysis_info(analysis_info),
+            outputs_prefix="Polygon.Analysis",
+            outputs_key_field="ID",
+            outputs=analysis_info,
+            indicator=domain,
+            raw_response=res
```
Comment (DeanArbel) [inline Packs/Polygon/Integrations/Polygon/Polygon.py:358]: Please omit analysis_info output from the indicator's output, as it's already being returned elsewhere.
These CommandResults should be just for the indicator and its relevant human readable.

File: Packs/Polygon/Integrations/Polygon/Polygon.py [hunk 3]
```diff
@@ -350,7 +368,15 @@ def get_network_indicators(report):
...
         )
-        ids.append(ip)
+        command_results.append(CommandResults(
+            readable_output=get_human_readable_analysis_info(analysis_info),
+            outputs_prefix="Polygon.Analysis",
+            outputs_key_field="ID",
+            outputs=analysis_info,
+            indicator=ip,
+            raw_response=res
+            )
```
Comment (DeanArbel) [inline Packs/Polygon/Integrations/Polygon/Polygon.py:378]: Please omit analysis_info output from the indicator's output, as it's already being returned elsewhere.
These CommandResults should be just for the indicator and its relevant human readable.

File: Packs/Polygon/Integrations/Polygon/Polygon.py [hunk 4]
```diff
@@ -361,13 +387,21 @@ def get_network_indicators(report):
...
-        ids.append(url)
-
-    return ids
+        command_results.append(CommandResults(
+            readable_output=get_human_readable_analysis_info(analysis_info),
+            outputs_prefix="Polygon.Analysis",
+            outputs_key_field="ID",
+            outputs=analysis_info,
+            indicator=url,
+            raw_response=res
```
Comment (DeanArbel) [inline Packs/Polygon/Integrations/Polygon/Polygon.py:396]: Please omit analysis_info output from the indicator's output, as it's already being returned elsewhere.
These CommandResults should be just for the indicator and its relevant human readable.

File: Packs/Polygon/Integrations/Polygon/Polygon.py [hunk 5]
```diff
@@ -377,25 +411,51 @@ def get_monitor_indicators(report):
...
             path=p.get('filename'),
         )
-        ids.append(process)
+        command_results.append(CommandResults(
+            readable_output=get_human_readable_analysis_info(analysis_info),
+            outputs_prefix="Polygon.Analysis",
+            outputs_key_field="ID",
+            outputs=analysis_info,
+            indicator=process,
+            raw_response=res
```
Comment (DeanArbel) [inline Packs/Polygon/Integrations/Polygon/Polygon.py:420]: Please omit analysis_info output from the indicator's output, as it's already being returned elsewhere.
These CommandResults should be just for the indicator and its relevant human readable.

File: Packs/Polygon/Integrations/Polygon/Polygon.py [hunk 6]
```diff
@@ -377,25 +411,51 @@ def get_monitor_indicators(report):
...
-    indicators += get_packages_indicators(report)
-    indicators += get_network_indicators(report)
-    indicators += get_monitor_indicators(report)
+                command_results.append(CommandResults(
+                    readable_output=get_human_readable_analysis_info(analysis_info),
+                    outputs_prefix="Polygon.Analysis",
+                    outputs_key_field="ID",
+                    outputs=analysis_info,
+                    indicator=reg,
+                    raw_response=res
```
Comment (DeanArbel) [inline Packs/Polygon/Integrations/Polygon/Polygon.py:435]: Please omit analysis_info output from the indicator's output, as it's already being returned elsewhere.
These CommandResults should be just for the indicator and its relevant human readable.

**Review by DeanArbel — CHANGES_REQUESTED** [review]
This is not a simple change, please see if you can test it after you address my comments below.

**Comment by orhovy** [comment]
@DeanArbel 
Ran both the updated integration and the old one and the context diff was empty.
The old integration:
old
The new version:
new

**Comment by orhovy** [comment]
@DeanArbel 
Ran both the updated integration and the old one and the context diff was empty.
The old integration:
old
The new version:
new_1

**Review by DeanArbel — APPROVED** [review]
Nice

**Pull request merged**